.github/2.0/cookbooks/Hub.md

```diff
@@ -0,0 +1,155 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+Table of Contents
+
+- [Hubble CLI Guidelines](#hubble-cli-guidelines)
+  - [Prerequisites](#prerequisites)
+  - [1. Create Executor](#1-create-executor)
+  - [2. Push and Pull CLI](#2-push-and-pull-cli)
+    - [2.1 Distribute your executor](#21-distribute-your-executor)
+    - [2.2 Pull distributed executor](#22-pull-distributed-executor)
+  - [3. Use in Jina Flow](#3-use-in-jina-flow)
+    - [3.1 Using docker images](#31-using-docker-images)
+    - [3.2 Using source codes](#32-using-source-codes)
+    - [3.3 Override Default Parameters](#33-override-default-parameters)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# JinaHub Guidelines
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/get-docker) installed.
+- `jina[standard] >= 2.0.0` installed.
+    - Run `pip install -U "jina[standard]>=2.0.0"` to upgrade jina to correct version.
+
+## 1. Create Executor
+
+To create your Executor, you just need to run this command in your terminal:
+
+```terminal
+jina hub new
+```
+
+A wizard will ask you some questions about the Executor. For the basic configuration, you will be asked two things: The Executor’s name and where it should be saved. For this tutorial, we will call ours RequestLogger. And you can save it wherever you want to have your project. The wizard will ask if you want to have a more advanced configuration, but it is unnecessary for this tutorial.
+
```

Comment:
I think it would be great if we inform users about what the command will generate and maybe provide some details about the the created files
```suggestion
After running the command, a project with the following structure will be generated:
```text
MyExecutor/
├── Dockerfile	        # Optional
├── manifest.yml
├── config.yml
├── README.md
├── requirements.txt
└── executor.py
```

Reply:
addressed